.travis.yml

```diff
@@ -106,19 +106,19 @@ before_install:
 install:
     - cd $girder_path
     - python setup.py develop
-    - pip install --upgrade 'git+https://github.com/cdeepakroy/ctk-cli'
-    - pip install -r $main_path/requirements.txt
+    - cd $large_image_path
+    - pip install -U -r requirements.txt
+    - pip install -e .
```

Comment:
Is there a reason you are installing large_image in development mode?  I've found that when in development mode, the HistomicsTK modules don't find the large_image modules unless you mess with PYTHONPATH.

Instead of these two lines, `python setup.py install` should be sufficient.